shared_objects/Makefile

```diff
@@ -0,0 +1,17 @@
+CC=$(HOME)/cheri/output/sdk/bin/riscv64-unknown-freebsd13-clang
+CFLAGS=-march=rv64imafdcxcheri -mabi=l64pc128d --sysroot=$(HOME)/cheri/output/rootfs-riscv64-hybrid -mno-relax -g
+SSHPORT=10021
+export
```

Comment:
I can see where?

Reply:
I assumed `export` on its own line wasn't supposed to be there? But maybe it does something I don't know about?!

Reply:
Oh yes it is supposed to make all the variables before it be stable from the env, like `CC=gcc make`.

Reply:
Is it needed here? I think it has the same effect with and without?

Reply:
The reason it is here is because I copied that from the root Makefile, which originally I wanted to be capable of running make in the subdirectories and pass the CC and CFLAGS from there. You are right it might not be needed.

Reply:
If it isn't needed, let's kill it, otherwise we'll keep copy and pasting it without ever understanding it :) If it's also unnecessary at the top level, you could do a separate PR which just kills it there too?